roughness-ref.csv

```diff
@@ -14,8 +14,8 @@ group,land cover,z0,zd,ra,reference
 7-1,,,,,
 7-2,,,,,
 8-1,,,,,
-8-2,,,,,
+8-2,Grasslands,2.6cm,10cm,0.4scm-1,"Evaporation from a Themeda Grassland. II. Resistance Model of Plant Evaporation"(Dunin et al 1978)
```

Comment:
Thanks for the update.
Let's modify it like this: "Dunin et al. (1978) <URL to this paper>"

Reply:
I'll correct it for you this time; but note in the future, please follow the academic style for such citations.